cli/cmd/plugin/ui/web/tanzu-ui/src/state-management/stores/Store.aws.tsx

```diff
@@ -2,30 +2,28 @@
 import React, { createContext, ReactNode, useReducer } from 'react';
 // App imports
 import { AWS_DEFAULT_VALUES } from '../../shared/constants/defaults/aws.defaults';
-import awsReducer from '../reducers/Wizard.reducer';
+import awsReducer from '../../views/providers/aws/Aws.reducer';
 import { STORE_SECTION_FORM } from '../reducers/Form.reducer';
 import { StoreDispatch } from '../../shared/types/types';
+import { STORE_SECTION_AWS_RESOURCES } from '../../views/providers/aws/AwsResources.reducer';
 
 const initialState = {
     [STORE_SECTION_FORM]: {
         // Auth - Credential Profile
         PROFILE: '',
 
-        // Auth - Temporary Credentials
-        SECRET_ACCESS_KEY: '',
-        SESSION_TOKEN: '',
-        ACCESS_KEY_ID: '',
+        // Region
```

Comment:
I think all of these hard-coded attributes would be better as references to the AWS_FIELDS enum:
`[AWS_FIELDS.REGION]` instead of simply `REGION`

Reply:
`REGION`  already exists as a field in enum `AWS_FIELDS` . 
And yes, I just moved them. I am gonna move it back to cancel this change which is not necessary